Detectors/EMCAL/workflow/src/EventBuilderSpec.cxx

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction
+
+#include <algorithm>
+#include <iterator>
+#include <unordered_map>
+#include <gsl/span>
+
+#include "DataFormatsEMCAL/Cell.h"
+#include "DataFormatsEMCAL/TriggerRecord.h"
+#include "Framework/InputSpec.h"
+#include "Framework/Logger.h"
+#include "EMCALWorkflow/EventBuilderSpec.h"
+
+using namespace o2::emcal;
+
+void EventBuilderSpec::init(framework::InitContext& ctx)
+{
+}
+
+void EventBuilderSpec::run(framework::ProcessingContext& ctx)
+{
+  auto dataorigin = o2::header::gDataOriginEMC;
+  std::unordered_map<o2::header::DataHeader::SubSpecificationType, gsl::span<const TriggerRecord>> triggers;
+  std::unordered_map<o2::header::DataHeader::SubSpecificationType, gsl::span<const Cell>> cells;
+
+  std::vector<Cell> outputCells;
+  std::vector<TriggerRecord> outputTriggers;
+
+  auto posCells = ctx.inputs().getPos("inputcells"),
+       posTriggerRecords = ctx.inputs().getPos("inputtriggers");
+  auto numSlotsCells = ctx.inputs().getNofParts(posCells),
+       numSlotsTriggerRecords = ctx.inputs().getNofParts(posTriggerRecords);
+  for (decltype(numSlotsCells) islot = 0; islot < numSlotsCells; islot++) {
+    auto celldata = ctx.inputs().getByPos(posCells, islot);
+    auto subspecification = framework::DataRefUtils::getHeader<header::DataHeader*>(celldata)->subSpecification;
+    // Discard message if it is a deadbeaf message (empty timeframe)
+    if (subspecification == 0xDEADBEEF) {
```

Comment:
see comment https://github.com/AliceO2Group/AliceO2/pull/7642/files#r752583346 
Why not sending with the standard subspec and skip if the payload is absent (which will happen if the raw data had 0xdeadbeef)?